packages/veritone-react-common/src/components/EngineOutputHeader/index.js

```diff
@@ -46,12 +61,86 @@ class EngineOutputHeader extends Component {
     }
   };
 
+  toggleIsMoreMenuOpen = () => {
+    this.setState(prevState => {
+      return {
+        isMoreMenuOpen: !{ ...prevState }.isMoreMenuOpen
```

Comment:
you shouldn't need to destructure prevState. You should just be able to do { isMoreMenuOpen: !prevState .isMoreMenuOpen }

Reply:
done